src/main/scala/zio/interop/java.scala

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.interop
+
+import _root_.java.nio.channels.CompletionHandler
+import _root_.java.util.concurrent.{ CompletableFuture, CompletionException, CompletionStage, Future }
+
+import zio._
+import zio.blocking.{ blocking, Blocking }
+
+import scala.concurrent.ExecutionException
+
+object java {
```

Comment:
@sideeffffect Eh, I'd avoid that. What if someone does `import zio.interop._` ? Then all subsequent `import java....` & `java.Smth` references will fail. It's the same reason we use `zio.interop.catz`, not `zio.interop.cats` – to avoid clashes with a root package.

Reply:
`javaz`?

Reply:
OK :D Although I think the old name isn't so bad either

Reply:
I wanted to get rid of the `concurrent` part, because in the future I would like this library to be interop for everything JRE 8, not just for `java.util.concurrent`

Reply:
OK :+1: